app/views/YourFinancialInstitutionsView.scala.html

```diff
@@ -23,17 +23,21 @@
         govukRadios: GovukRadios,
         govukButton: GovukButton,
         govukSummaryList: GovukSummaryList,
-        heading: Heading
+        heading: Heading,
+        notification: Notification
 )
 
-@(form: Form[_], institutions: SummaryList)(implicit request: Request[_], messages: Messages)
+@(form: Form[_], institutions: SummaryList, removedInstitutionName: Option[String] = None)(implicit request: Request[_], messages: Messages)
 
 @layout(pageTitle = title(form, messages("yourFinancialInstitutions.title"))) {
 
     @if(form.errors.nonEmpty) {
         @govukErrorSummary(ErrorSummaryViewModel(form))
     }
 
+    @if(removedInstitutionName.isDefined) {
+        @notification(messages("yourFinancialInstitutions.notification",removedInstitutionName.get))}
```

Comment:
a nicer way to avoid using `.get`
```
  @removedInstitutionName.foreach { name => 
        @notification(messages("yourFinancialInstitutions.notification",name))        
  }
  ```
  
  just aesthetic - not necessary

Reply:
i tried using this and it doesn't seem work - looks like it should though